keyboards/gowla/config.h

```diff
@@ -0,0 +1,38 @@
+/*
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
```

Comment:
This is just a food for thoughts and not a blocker, but if you intend to add support for more of your designs, I recommend creating a vendor directory and putting the board sources there.

Like `keyboards/gowla/macro_board` or something similar.

Reply:
Thanks for your suggestion. I think I will keep this in mind and make this change once/if I create any new designs. Which might not be for a few weeks :)